tests/spec/fs.open.spec.js

```diff
@@ -136,4 +136,13 @@ describe('fs.open', function() {
       });
     });
   });
+
+  it('should return an error when flag in invalid', function(done) { 
+    var fs = util.fs();
+
+    expect(fs.open('/myfile', 'abcd', function(){
+      done();
```

Comment:
I don't think we need `done` at all in this test, since nothing async is happening (i.e., it throws right away